### PR TITLE
TomoChain (TOMO) Rebranding to Viction (VIC)

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

**Comment by defisaur** [comment]
Added to https://github.com/trustwallet/assets/pull/26921

File: blockchains/viction/assets/0x567b1e373f7285e9a32f5eeB9305b6964B6a3532/logo.png
Comment (Gb2real) [inline blockchains/viction/assets/0x567b1e373f7285e9a32f5eeB9305b6964B6a3532/logo.png:1]: ok

**Review by Gb2real — COMMENTED** [review]
ok

**Review by Gb2real — COMMENTED** [review]
ok

**Review by 777668z0 — COMMENTED** [review]
تم

**Pull request closed without merging**